### ISSUE TITLE
[feature request] Playlist2Download

### ISSUE THREAD

**Issue by alafad** [PlexRipper/PlexRipper#188]
Hello, a feature would be great that can read out my plex Watch/Playlist, which downloads all media from these .

Is this possible ?

Thank you for the great project

**Comment by JasonLandbridge** [comment]
Hi there, 

I'm not sure what you mean, do you mean that you have a playlist on a server that you would like to download with PlexRipper?

Cheers!

**Comment by alafad** [comment]
> Hi there, 
> 
> I'm not sure what you mean, do you mean that you have a playlist on a server that you would like to download with PlexRipper?
> 
> Cheers!

Yes this would be an fantastic feature so you can use the plex UI to search an actor and dumb download all selected entries in a playlist.

**Comment by JasonLandbridge** [comment]
Ah I see, I will keep it in mind but before I can add that I will need to add search first. Thanks for the suggestion!
